content/Community/committee.en.md

```diff
@@ -31,10 +31,17 @@ The DSC Resource Kit was already operating under similar structure, but it wasn'
 
 ## Initial Members
 
+<a href="https://github.com/kwirkykat" target="_blank"><img src="../../images/community/Katie.jpeg" alt="drawing" style="width:100px; display:block; float: right; margin-left: auto;margin-right:auto; padding-left: 0px; border-radius:50%" /></a>
```

Comment:
Instead of uploading the images, maybe we can link to the GiHub URL? For example for Daniel; https://avatars0.githubusercontent.com/u/7589164?s=400&v=4. But it would probably break when changing image, but keeping it an URL, then it possible for the member to use any avatar site to publish the image and link to that URL.

Reply:
Click on each avatar image on GitHub to get that URL by the way. :)

Reply:
Yeah, I found that but decided against.
Given the link, I thought it would break, and that does not really gives us the chance to override it (I'd rather have people's face than just a character or random pic...).
We should probably check if the links stays after the pic is changed, and if that's true then we could replace the source.

Maybe improvement for later though...

Reply:
Just verified this, and the GitHub image URL stays the same after image gets updated.

Reply:
Should be done ;)